packages/esm-service-queues-app/src/add-provider-queue-room/add-provider-queue-room.resource.ts

```diff
@@ -2,7 +2,7 @@ import { openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
 import useSWR from 'swr';
 import { type ProvidersQueueRoom, type QueueRoom } from '../types';
 
-export function useQueueRooms(location: string, queueUuid: string) {
+export function useQueueRooms(location: string, queueUuid: string = '') {
```

Comment:
This technically works, but we should use `null` to denote absence of a value instead of empty string.